qemu/tests/cfg/x86_cpu_model.cfg

```diff
@@ -202,13 +203,15 @@
                 flags = ""
         - Icelake-Client:
             only HostCpuVendor.intel
+            no RHEL.9
```

Comment:
Is this also related to Host_RHEL?

Reply:
> Is this also related to Host_RHEL?

The original qemu bug is for virtualization part and we have no kernel bug to disble these platforms so the host should boot up successfully.